SPECS-EXTENDED/cassandra/cassandra.spec

```
@@ -0,0 +1,209 @@
+%global __os_install_post %{nil}
+%global debug_package %{nil}
+
+Summary:        Cassandra is a highly scalable, eventually consistent, distributed, structured key-value store
+Name:           cassandra
+Version:        4.0.3
+Release:        1%{?dist}
+URL:            http://cassandra.apache.org/
+License:        Apache License, Version 2.0
+Group:          Applications/System
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        http://archive.apache.org/dist/cassandra/%{version}/apache-%{name}-%{version}-src.tar.gz
+Source1:        cassandra.service
+Source2:        cassandra-build-cache-4.0.3.tar.gz
+ExclusiveArch:  x86_64
+
+BuildRequires:  ant
+BuildRequires:  ant-junit
+BuildRequires:  ant-jmf
+BuildRequires:  ant-scripts
+BuildRequires:  unzip
+BuildRequires:  zip
+BuildRequires:  wget
+BuildRequires:  git
+BuildRequires:  tar
+BuildRequires:  which
+BuildRequires:  systemd-rpm-macros
+BuildRequires:  msopenjdk-11
+BuildRequires:  javapackages-local-bootstrap
+BuildRequires:  javapackages-tools
+BuildRequires:  xml-commons-apis
+BuildRequires:  xerces-j2
+BuildRequires:  python3
+BuildRequires:  python3-devel
+BuildRequires:  python3-libs
+BuildRequires:  python3-setuptools
+
+Requires:       msopenjdk-11
+Requires:       javapackages-tools
+Requires:       which
+Requires:       gawk
+Requires:       shadow-utils
+Requires(post): /bin/chown
+%{?systemd_requires}
+
+%description
+Cassandra is a highly scalable, eventually consistent, distributed, structured key-value store.
+Cassandra brings together the distributed systems technologies from Dynamo and the log-structured storage engine from Google's BigTable.
+
+%prep
+%autosetup -p1 -n apache-%{name}-%{version}-src
+
+%build
+# refer to Readme file for 
+# cassandra-build-cache
+cp %{SOURCE2} .
+tar -xvf  %{SOURCE2}
+mkdir -p ~/.m2
+mv repository ~/.m2/
+
+export JAVA_HOME=$(echo /usr/lib/jvm/msopenjdk-11)
```

Please use `%{java_home}` instead.

```suggestion
export JAVA_HOME="%{java_home}"
```

---

updated